### PR TITLE
remove old implementation of sidecars

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- remove old implementation of Sidecars
- adjust tests to extensibility implementation
- add workaround for tests `findLastIndex`
- add function for cypress to mock extension

**Related issue(s)**
part of #1964

### CODE REVIEW

**Review by akucharska — CHANGES_REQUESTED** [review]
Remove sidecars from translations

**Review by akucharska — DISMISSED** [review]

File: core-ui/src/components/Extensibility/hooks/useVariables.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ import * as jp from 'jsonpath';
 import { jsonataWrapper } from '../helpers/jsonataWrapper';
 import { VarStoreContext } from '../contexts/VarStore';
 
+// Cypress uses chorme 95 which doesn't support findLastIndex
```
Comment (IwonaLanger) [inline core-ui/src/components/Extensibility/hooks/useVariables.js:8]:
```suggestion
// Cypress uses Chrome 95 which doesn't support findLastIndex
```

**Review by IwonaLanger — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**